config.go

```diff
@@ -158,16 +158,16 @@ func (cfg *Config) CacheConfigLoad(path string) error {
 	return nil
 }
 
-// Configures whether generated wasm code will be interrupted via interrupt
+// SetInterruptable configures whether generated wasm code will be interrupted via interrupt
```

Comment:
I think we can change this from `will be interrupted` to `can be interrupted` for clarity. Thoughts?